docs/interpreter/springxd.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: page
+title: "SpringXD Interpreter"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+
+## SpringXD  Interpreter for Apache Zeppelin
+
+### Overview
+[Spring XD](http://docs.spring.io/spring-xd/docs/current/reference/html/#_reference_guide) is a unified, distributed, and extensible service for data ingestion, real time analytics, batch processing, and data export. SpringXD helps to build big-data applications that integrate many disparate systems into one cohesive solution across a range of use-cases. 
+SpringXD supports two processing abstractions: (1) [Streams](http://docs.spring.io/spring-xd/docs/current/reference/html/#streams) for event driven data, and (2) [Jobs](http://docs.spring.io/spring-xd/docs/current/reference/html/#jobs) like MR, PIG, Hive, Cascading, SQL and so on, for batch data types. Spring XD tries to blur the boundaries between the two using a channel abstraction, so that, for example, a stream can trigger a batch job, and a batch job can send events and, in turn, trigger other streams. An expressive DSL is provided for defining streams and jobs: 
+
+```
+myStream = http | filter --expression=payload='foo' | log
+```
+
+Zeppelin provides interpreters for SpringXD `Streams` and `Jobs`: 
+
+<table class="table-configuration">
+  <tr>
+    <th>Name</th>
+    <th>Class</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>%xd.stream</td>
+    <td>SpringXdStreamInterpreter</td>
+    <td>Provides an environment to create SpringXD Streams - defines the ingestion of event driven data from a `source` to a sink that passes through any number of `processors. </td>
```

Comment:
I think that back quotation marks are not working in the table tag. Like this below image. 
![screen shot 2016-01-18 at 5 53 55 pm](https://cloud.githubusercontent.com/assets/10060731/12387103/72e98b1a-be0d-11e5-914c-7c4a88d8e513.png)

So, why don't you use `<code>` tag instead of the back quotations? : )